css/index.css

```diff
@@ -0,0 +1,8 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap%27");
```

Comment:
The work is **Great** here, but there is a principle in software designing says `don't repeat yourself`
* don't repeat the code when there is no need to do that. As you can see here in the all `CSS` files, you repeat the same code more than 3 times, and in fact, you shouldn't do that because you can simply create a `General CSS File` contains all general styles, like font declaration, styles within `*` tag, and another general styles, and you can put it into All html files you need, so here you avoid repeating, and you collect all the codes in one place.